src/solc_0.8/defi/rules/ContributionRules.sol

```diff
@@ -60,9 +65,30 @@ contract ContributionRules is Ownable {
         uint256 multiplierERC721 = multiplierBalanceOfERC721(account);
         uint256 multiplierERC1155 = multiplierBalanceOfERC1155(account);
 
+        // check if the calculated multipliers exceeds the limit
+        if (erc721MultiplierLimit < multiplierERC721) {
```

Comment:
erc721MultiplierLimit > ERC721MultiplierLimit

Reply:
updated to multiplierLimitERC271 -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/627/commits/b391428f65f735bf5b88789e0b9702049934fb4b